### PR TITLE
Set opaque background on markup and images

### PR DESCRIPTION
**Pull request by silverwind**
- Set opaque background on markup images so they can visually break `<hr>`
- Change padding of comment box so `padding` is provided by the `.markup` element instead of its parent, matching the file rendering view which does the same.

Before:
Screenshot 2023-03-19 at 19 22 03

After:
Screenshot 2023-03-19 at 19 23 26
image

Example documents:
https://try.gitea.io/silverwind/symlink-test/src/branch/master/test-page.md
https://github.com/silverwind/symlink-test/blob/master/test-page.md

### CODE REVIEW

**Review by yardenshoham — APPROVED** [review]

**Comment by silverwind** [comment]
There is a small regression in dropzone attachment rendering because of the padding change, checking...

**Comment by silverwind** [comment]
Regression fixed. Dropzone `<hr>` now also matches the reaction `<hr>` in width which it previously didn't.

Screenshot 2023-03-19 at 19 42 33

**Review by zeripath — APPROVED** [review]

**Comment by jolheiser** [comment]
🎺 🤖

File: web_src/css/markup/content.css [hunk 1]
```diff
@@ -3,6 +3,7 @@
   font-size: 16px;
   line-height: 1.5 !important;
   word-wrap: break-word;
+  background: var(--color-box-body);
```
Comment (wxiaoguang) [inline web_src/css/markup/content.css:6]: I guess it should be `inherit`, then we do not need to fix various colors again and again.

Or just remove it? I can't see difference without this style.


image

Comment (silverwind) [reply web_src/css/markup/content.css:6]: `background` does not inherit or cascade by default, so `inherit` would only inherit from the direct element parent, but in the release list we actually would want to inherit from `body`.

File: web_src/css/repository.css [hunk 2]
```diff
@@ -2740,6 +2740,12 @@
   width: 100% !important;
   max-width: 100% !important;
   margin: 0 !important;
+  padding: 0 !important;
```
Comment (wxiaoguang) [inline web_src/css/repository.css:2743]: I would also suggest to revert this change.

https://github.com/go-gitea/gitea/pull/23750#issuecomment-1486249539

It causes unnecessary overwriting problems.
